### ISSUE TITLE
TypeTray - inner walls with different thickness

### ISSUE THREAD

**Issue by Blancanegra** [florianfesti/boxes#109]
As now is possible to generate beautiful boxes with divisions, in some cases would be dessirable to use a thin material for the inner sections.

For example, a big box with 5 mm thickness for exterior walls and 3 mm for interior walls and floor, etc.

**Comment by florianfesti** [comment]
Yeah, that's a cool feature. Boxes.py can actually do that as it can instantiate the edges with different settings. It also will require adjusting the measurement calculations to use the two different material thicknesses.
